amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -606,37 +597,6 @@ describe('SearchPage', () => {
     });
   });
 
-  describe('renderPopularTables', () => {
```

Comment:
nit: What is correct to do here is to update the test for `renderPopularTables` as opposed to removing it.`renderPopularTables` is still a part of `SearchPage` and should still be tested. It would just have to two simple checks to test that a `BookmarkList` and a `PopularTables` is rendered. However this is a nit simply because I understand it would be throw-away work based on upcoming PRs.

Reply:
Oh gotcha, yeah that makes sense. Will probably add the tests anyways as I can just move them over to the home page tests.